front/app/components/admin/NavigationTabs/Tab.tsx

```diff
@@ -72,22 +80,37 @@ type TabProps = {
   active: boolean;
   badge?: React.ReactNode;
   handleClick?: (event: MouseEvent<HTMLAnchorElement>) => void;
+  disabled?: boolean;
+  disabledTooltipText?: string;
 };
 
 const Tab = ({
   label,
   url,
-  active,
   badge,
   handleClick,
+  disabled,
+  disabledTooltipText,
   ...props
 }: TabProps) => (
-  <Container active={active} {...props}>
-    <Link to={url} onClick={handleClick}>
-      {label}
-      {badge && <>{badge}</>}
-    </Link>
-  </Container>
+  <Tippy
+    interactive={false}
+    placement="bottom"
+    theme={''}
+    disabled={!disabled}
+    maxWidth={350}
+    content={
+      <TooltipContentWrapper tippytheme="light">
+        {disabledTooltipText}
```

Comment:
Is there any case where `disabled` is `true`, but we don't want a `disabledTooltipText`? Maybe we only need `disabledTooltipText`?

Reply:
> Is there any case where disabled is true, but we don't want a disabledTooltipText?

No.

> Maybe we only need disabledTooltipText?

It might be a good idea indeed.